modules/svg/lines.js

```diff
@@ -94,18 +94,16 @@ export function svgLines(projection, context) {
 
 
         function waystack(a, b) {
+            // if top layer set, make top layer lines last to render.
+            var topLayer = Hoot.layers.getTopLayer();
+            if (topLayer) { return +a.mapId === topLayer ? 1 : -1 }
```

Comment:
when we are setting drawing order, use d3 / data array comparator function to make so the top layer map items are put at bottom of list and therefore rendered last.